### PR TITLE
Code Refactored

### PR DESCRIPTION
**Pull request by gopal-mahajan**
PR includes :
- changes in status code and response
- Readme and Setup  file updated 
- Code Refactored

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -90,7 +90,7 @@ pipeline {
     stage('Push Image') {
       when{
         expression {
-          return env.GIT_BRANCH == 'origin/master';
+          return env.GIT_BRANCH == 'origin/v3.5.0';
```
Comment (karun-singh) [inline Jenkinsfile:93]:
```suggestion
          return env.GIT_BRANCH == 'origin/master';
```
This needs to reflect the correct branch which would be master in this case.

**Comment by kailash** [comment]
ok to test

**Comment by karun-singh** [comment]
@gopal-mahajan Need to update the collection name in the Jenkinsfile here as well.

File: Jenkinsfile [hunk 2]
```diff
@@ -54,7 +54,7 @@ pipeline {
     stage('Run GIS interface server'){
       steps{
         script{
-            sh 'scp src/test/resources/IUDX-GIS-SERVER.postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
+            sh 'scp src/test/resources/IUDX_GIS_Server_APIs(v3.5.0).postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
```
Comment (karun-singh) [inline Jenkinsfile:57]:
```suggestion
            sh 'scp src/test/resources/IUDX_GIS_Server_APIs\(v3.5.0\).postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
```
need to escape the parentheses

File: Jenkinsfile [hunk 3]
```diff
@@ -67,7 +67,7 @@ pipeline {
           script{
             startZap ([host: 'localhost', port: 8090, zapHome: '/var/lib/jenkins/tools/com.cloudbees.jenkins.plugins.customtools.CustomTool/OWASP_ZAP/ZAP_2.11.0'])
             sh 'curl http://127.0.0.1:8090/JSON/pscan/action/disableScanners/?ids=10096'
-            sh 'HTTP_PROXY=\'127.0.0.1:8090\' newman run /var/lib/jenkins/iudx/gis/Newman/IUDX-GIS-SERVER.postman_collection.json -e /home/ubuntu/configs/gis-postman-env.json --insecure -r htmlextra --reporter-htmlextra-export /var/lib/jenkins/iudx/gis/Newman/report/report.html --reporter-htmlextra-skipSensitiveData'
+            sh 'HTTP_PROXY=\'127.0.0.1:8090\' newman run /var/lib/jenkins/iudx/gis/Newman/IUDX_GIS_Server_APIs(v3.5.0).postman_collection.json -e /home/ubuntu/configs/gis-postman-env.json --insecure -r htmlextra --reporter-htmlextra-export /var/lib/jenkins/iudx/gis/Newman/report/report.html --reporter-htmlextra-skipSensitiveData'
```
Comment (karun-singh) [inline Jenkinsfile:70]:
```suggestion
            sh 'HTTP_PROXY=\'127.0.0.1:8090\' newman run /var/lib/jenkins/iudx/gis/Newman/IUDX_GIS_Server_APIs\(v3.5.0\).postman_collection.json -e /home/ubuntu/configs/gis-postman-env.json --insecure -r htmlextra --reporter-htmlextra-export /var/lib/jenkins/iudx/gis/Newman/report/report.html --reporter-htmlextra-skipSensitiveData'
```
need to escape the parentheses

**Review by karun-singh — CHANGES_REQUESTED** [review]

**Review by karun-singh — APPROVED** [review]

**Comment by kailash** [comment]
retest this please

**Review by kailash — APPROVED** [review]

**Pull request merged**